docs/user-guide/nextcloud-integration/README.md

```diff
@@ -120,12 +119,36 @@ When a Nextcloud file or folder is linked to a work package, an OpenProject user
 
 However, all available actions depend on permissions the OpenProject user (or more precisely, the Nextcloud account tied to that user) has in Nextcloud. In other words, a user who does not have the permission to access the file in Nextcloud will also *not* be able to open, download, modify or unlink the file in OpenProject.
 
+## Disconnecting your Nextcloud and OpenProject accounts
+
+In certain cases, you may wish to disconnect your Nextcloud and OpenProject accounts. You might want to do this, for example, if you have multiple accounts and realise that you connected the wrong ones. 
+
+Since OAuth credentials were generated by both Nextcloud and OpenProject when you connected your accounts, we recommend disconnecting on both ends.
+
+#### In OpenProject
+
+In OpenProject, click on your avatar on the top right corner and navigate to ***My account** → **Access tokens***. 
+
+![OpenProject account settings page](my_account_access_tokens.png)
+
+You will see a list of all access tokens associated with your account, including for a Nextcloud OAuth application. Click on **Revoke** and, in the confirmation dialog that appears, click **OK**.
```

Comment:
So, this section will then change in my next PR I guess?

Reply:
Exactly.